config/hardware/addons/lights/caselight.cfg

```diff
@@ -0,0 +1,26 @@
+#####################################################################
+#   LED Control
+#####################################################################
+
```

Comment:
Can you add here the following lines to include the correct macros only when this caselight file is included in the config ? Today this file is only included when status_led is included and it's not good:
```
# Also include directly the leds control macros from here
[include ../../../../macros/addons/hardware_functions/status_leds.cfg]
```

Reply:
Ok done